src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs

```diff
@@ -71,7 +72,7 @@ protected Task<Solution> GetSolutionAsync(PinnedSolutionInfo solutionInfo, Cance
         protected async ValueTask<T> RunServiceAsync<T>(Func<CancellationToken, ValueTask<T>> implementation, CancellationToken cancellationToken)
         {
             WorkspaceManager.SolutionAssetCache.UpdateLastActivityTime();
-            using var _ = LinkToken(ref cancellationToken);
+            using var combined = cancellationToken.CombineWith(ClientDisconnectedSource.Token);
```

Comment:
`combined` is not used? Isn't this a no-op?

Reply:
➡️ See https://github.com/dotnet/roslyn/pull/48179#discussion_r497159268